html/elements/link.json

```
@@ -499,6 +499,57 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "secure_context_only": {
+            "__compat": {
+              "description": "Secure context only",
```

And then this would be…

```suggestion
              "description": "Secure context required",
```